_data/navigation.js

```diff
@@ -41,9 +41,11 @@ module.exports = {
       {
         section: 'ORACLE JOBS',
         contents: [
-          { url: '/docs/job-specifications/', title: 'Job Specifications' },
-          { url: '/docs/adapters/', title: 'Core Adapters' },
-          { url: '/docs/initiators/', title: 'Initiators' },
+          { url: '/docs/jobs/', title: 'Jobs' },
+          { url: '/docs/tasks/', title: 'Tasks' },
+          { url: '/docs/job-specifications/', title: 'Job Specifications [DEPRECATED]' },
```

Comment:
I think there's a better way we can structure the deprecated pages. Maybe version 1 and 2 parent pages, with 2 being the default?

Reply:
What is a parent page?

Reply:
Here's an example of one: https://docs.chain.link/docs/reference-contracts/

Take a look at the menu item, it acts as a parent to sub-pages.